.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Support request
+about: Help, something isn't working and I'm stuck!
+title: ''
+labels: 'status: needs triage, type: support'
+assignees: ''
+
+---
+
+<!--
+Before asking for support, please take a look at the [troubleshooting guide](https://haskell-language-server.readthedocs.io/en/latest/troubleshooting.html).
+This explains some common issues and will also help you to find the information that the issue template asks for.
+
+When filing a support ticket, please fill out as much of the information below as you can.
+This helps us to debug your issue, but is not required!
+-->
+
+## Your environment 
+
+Which OS do you use?
+<!-- Windows, MacOS, Ubuntu, ArchLinux, etc... -->
+Which LSP client (editor/plugin) do you use?
+<!-- Neovim+LanguageClient-neovim, emacs+lsp-mode, VS Codium+vscode-haskell, etc... -->
+Which version of HLS do you use and how did you install it?
+<!-- 1.7.0.1 from ghcup, etc. -->
+Which version of GHC do you use and how did you install it?
+<!-- 9.0.2 from stack/ghcup, etc. -->
+How is your project set up (alternative: link to the project)?
+<!-- stack.yaml, package.yaml, *.cabal files, cabal.project, hie.yaml -->
```

Comment:
I'd almost argue `hie.yaml` should be more prominent on its own line. Lack of this file is a major source of confusion.

Reply:
Well, although it continues to be true that most people shouldn't need it. But if they do have it they should definitely mention it! Perhaps I'll split it into "how is your project built?" and "have you configured HLS with hie.yaml?" or something?

Reply:
Yeah